### PR TITLE
Init file structure

### PR DESCRIPTION
**Pull request by steph-j**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,3 +36,5 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+static
```
Comment (don-smith) [inline .gitignore]: I think we're going to need this because we're using the repo to deploy to Heroku.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
+
 # stupid-questions
-An app for students to anonymously ask "stupid" questions
+
+This is a small sample project you can use to start a new React/Redux app. It contains:
+
```
Comment (don-smith) [inline README.md:5]: Can we please update to it is relevant to this project, rather than the starter.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    "react-dom": "^15.1.0",
+    "webpack": "^1.13.1",
+    "webpack-dev-server": "^1.14.1"
+  },
+  "dependencies": {
+    "body-parser": "^1.15.1",
+    "cors": "^2.7.1",
+    "express": "^4.13.4",
+    "react-redux": "^4.4.5",
+    "redux": "^3.5.2",
```
Comment (don-smith) [inline package.json:60]: May as well add `react-router` from the start.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    "babel-loader": "^6.2.4",
+    "babel-polyfill": "^6.9.1",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-react-hmre": "^1.1.1",
+    "copy-webpack-plugin": "^3.0.1",
+    "enzyme": "^2.3.0",
+    "node-dev": "^3.1.3",
+    "node-inspector": "^0.12.8",
+    "react": "^15.1.0",
```
Comment (don-smith) [inline package.json]: This should be moved to a dependency instead of a dev-dependency.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "redux-starter",
+  "version": "1.0.0",
+  "description": "A starter React/React solution",
```
Comment (don-smith) [inline package.json]: The `name` and `description` should be updated.

File: server/server.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+var cors = require('cors')
+var path = require('path')
+
+var things = require('./routes/things')
+
+var PORT = process.env.PORT || 3000
+var app = express()
+
+app.use(bodyParser.json())
+app.use(cors({origin: 'http://localhost:8080'}))
```
Comment (don-smith) [inline server/server.js]: I don't think we need the CORS setup.

File: .gitignore (hunk 1 above)
Comment (steph-j) [inline .gitignore]: removed static from the .gitignore.

File: README.md (hunk 2 above)
Comment (steph-j) [inline README.md:5]: Done

File: package.json (hunk 3 above)
Comment (steph-j) [inline package.json:60]: Added

File: package.json (hunk 4 above)
Comment (steph-j) [inline package.json]: Done

File: package.json (hunk 5 above)
Comment (steph-j) [inline package.json]: Done

File: server/server.js (hunk 6 above)
Comment (steph-j) [inline server/server.js]: Taken out

File: package.json [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    "enzyme": "^2.3.0",
+    "node-dev": "^3.1.3",
+    "node-inspector": "^0.12.8",
+    "react-addons-test-utils": "^15.1.0",
+    "webpack": "^1.13.1",
+    "webpack-dev-server": "^1.14.1"
+  },
+  "dependencies": {
+    "body-parser": "^1.15.1",
+    "cors": "^2.7.1",
```
Comment (don-smith) [inline package.json]: We don't need this any longer.

Comment (steph-j) [inline package.json]: Done

**Pull request merged**